common/controlplaneconfig.h

```diff
@@ -104,6 +105,19 @@ class interface_t
 	common::globalBase::tFlow flow;
 };
 
+class bird_import_t
+{
+public:
+	SERIALIZABLE(socket, vrf);
+
+public:
+	inline static const std::string socketStr = "socket";
+	inline static const std::string vrfStr = "vrf";
+
+	std::string socket;
+	std::string vrf;
+};
```

Comment:
Since everything is public, why not just 
```suggestion
struct bird_import_t
{
	std::string socket;
	std::string vrf;
	
	static constexpr auto socketStr = "socket";
	static constexpr auto vrfStr = "vrf";
	
	SERIALIZABLE(socket, vrf);
};
```

Reply:
I agree, but I followed the standard of writing classes in this file.